src/main/msp/msp.c

```diff
@@ -1116,6 +1117,15 @@ static bool mspProcessOutCommand(mspDescriptor_t srcDesc, int16_t cmdMSP, sbuf_t
             // config state flags - bits to indicate the state of the configuration, reboot required, etc.
             // other flags can be added as needed
             sbufWriteU8(dst, (getRebootRequired() << 0));
+
+            // Added in API version 1.46
+            // Write CPU temp
+#ifdef USE_ADC_INTERNAL                
+            int16_t coretemp = getCoreTemperatureCelsius();
+#else
+            int16_t coretemp = 0;
+#endif                
+            sbufWriteU16(dst, coretemp);
```

Comment:
We usually do the ifdef with the sbufWrite directly, I think is cleaner:
```suggestion
            // Added in API version 1.46
            // Write CPU temp
#ifdef USE_ADC_INTERNAL                
            sbufWriteU16(dst, getCoreTemperatureCelsius());
#else
            sbufWriteU16(dst, 0);
#endif
```
But I don't know if we have some rule about it.

Reply:
Well, it's my debug style, possible to inspect return values, but since I can't debug, then :-)